st2common/st2common/util/loader.py

```diff
@@ -259,3 +259,21 @@ def load_meta_file(file_path):
 
     with open(file_path, 'r') as f:
         return PARSER_FUNCS[file_ext](f)
+
+
+def get_available_plugins(namespace):
+    """Return names of the available / installed plugins for a given namespace.
+    """
+    from stevedore.extension import ExtensionManager
+
+    manager = ExtensionManager(namespace=namespace, invoke_on_load=False)
+    return manager.names()
+
+
+def get_plugin_instance(namespace, name):
```

Comment:
Perhaps we should move and re-use the similarly named method we use from auth backends code here and re-use it in both places.

Also, right now the code just returns the driver class and not the instance :)

Reply:
I agree with @Kami here. At first glance, I thought this method is returning an instance. But since it's invoke_on_load=False, it's just returning the class.